src/main/java/core/basesyntax/model/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.model;
+
+public class Rectangle extends Color {
+    private double length;
+    private double width;
+
+    public Rectangle(String color, double width, double length) {
+        super(color);
+        this.width = width;
+        this.length = length;
+    }
+
+    @Override
+    public double getArea() {
+        return length * width;
```

Comment:
The method getArea() is conceptually different from the method draw(). According to the checklist, they should not be in the same class. Consider creating a separate interface for area calculation.